recipes/_configure_shc_alerts.rb

```diff
@@ -0,0 +1,18 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: _configure_shc_alerts
+#
+# Configures the alert settings for the search heads in a search head cluster
+
+hash = CernerSplunk::DataBag.load node['splunk']['config']['alerts'], pick_context: ['shc_configs']
+
+unless hash
+  Chef::Log.info 'Splunk Alerts not configured for the search heads in the search head cluster.'
+  return
+end
+
+splunk_template 'shcluster/deployer-configs/alert_actions.conf' do
```

Comment:
I remember we had a discussion about changing the name of the app to _shcluster so that 1) its configs will have higher precedence 2) will match the _cluster app that we have in the indexer cluster.